File: Latihan soal b/php/TimSepakbola.php
<?php

/* membuat class */
class TimSepakbola{

	/* deklarasi atribut private */
	private $nama = "";
	private $negara = "";
	private $tahun = 0;
	private $pemain = "";
	private $manajer = "";
	private $stadion = "";

	/* constructor */
	public function __construct($nama = "", $negara = "", $tahun = 0, $pemain = "", $manajer = "", $stadion = ""){
		$this->nama = $nama;
		$this->negara = $negara;
		$this->tahun = $tahun;
		$this->pemain = $pemain;
	}

	/* setter dan getter untuk atribut nama */
	public function setNama($nama){
		$this->nama = $nama;
	}

	public function getNama(){
		return $this->nama;
	}

	/* setter dan getter untuk atribut negara */
	public function setNegara($negara){
		$this->negara = $negara;
	}

	public function getNegara(){
		return $this->negara;
	}

	/* setter dan getter untuk atribut tahun */
	public function setTahun($tahun){
		$this->tahun = $tahun;
	}

	public function getTahun(){
		return $this->tahun;
	}

	/* setter dan getter untuk atribut pemain */
	public function setPemain($pemain){
		$this->pemain = $pemain;
	}

	public function getPemain(){
		return $this->pemain;
	}

	/* setter dan getter untuk atribut manajer */
	public function setManajer($manajer){
		$this->manajer = $manajer;
	}

	public function getManajer(){
		return $this->manajer;
	}

	/* setter dan getter untuk atribut stadion */
	public function setStadion($stadion){
		$this->stadion = $stadion;
	}

	public function getStadion(){
		return $this->stadion;
	}
}

?>

File: Latihan soal b/java/TimSepakbola.java
/* membuat class */
public class TimSepakbola{
	
	/* deklarasi atribut private */
	private String nama;
	private String negara;
	private int tahun;
	private String pemain;
	private String manajer;
	private String stadion;

	/* constructor */
	public TimSepakbola(){}

	public TimSepakbola(String nama, String negara, int tahun, String pemain, String manajer, String stadion){
		this.nama = nama;
		this.negara = negara;
		this.tahun = tahun;
		this.pemain = pemain;
		this.manajer = manajer;
		this.stadion = stadion;
	}

	/* setter dan getter untuk atribut nama */
	public String getNama(){
		return nama;
	}

	public void setNama(String nama){
		this.nama = nama;
	}

	/* setter dan getter untuk atribut negara */
	public String getNegara(){
		return negara;
	}

	public void setNegara(String negara){
		this.negara = negara;
	}

	/* setter dan getter untuk atribut tahun */
	public int getTahun(){
		return tahun;
	}

	public void setTahun(int tahun){
		this.tahun = tahun;
	}

	/* setter dan getter untuk atribut pemain */
	public String getPemain(){
		return pemain;
	}

	public void setPemain(String pemain){
		this.pemain = pemain;
	}

	/* setter dan getter untuk atribut manajer */
	public String getManajer(){
		return manajer;
	}

	public void setManajer(String manajer){
		this.manajer = manajer;
	}

	/* setter dan getter untuk atribut stadion */
	public String getStadion(){
		return stadion;
	}

	public void setStadion(String stadion){
		this.stadion = stadion;
	}
}